layouts/partials/head.html

```diff
@@ -0,0 +1,54 @@
+<meta charset="utf-8">

```

Comment:
As I said in [this message](https://github.com/PrestaShop/ps-org-theme/pull/1#discussion_r786902952), I think that the templates should be kept in the theme repo, and only the overrides and the images here.